web/templates/policy/coc.html.md

```diff
@@ -0,0 +1,93 @@
+## Code of Conduct
+
+Hex exists to facilitate sharing code, by making it easy for Erlang ecosystem developers to publish and distribute packages.
+
+Hex is a piece of technology, but more importantly, it is a community.
+
+We believe that our mission is best served in an environment that is friendly, safe, and accepting; free from intimidation or harassment.
+
+Towards this end, certain behaviors and practices will not be tolerated.
+
+Hex is the set of formats and APIs that makes up the specification of Hex. Hex.pm is an implementation of the Hex specifications, it is comprised of this website and associated HTTP API, the package repository including packages published to Hex.pm. In this document Hex and Hex.pm is used interchangeably but it refers to Hex.pm.
+
+### tl;dr
+
+  * Be respectful.
+  * We're here to help: [support@hex.pm](mailto:support@hex.pm)
```

Comment:
Minor note: I believe `<support@hex.pm>` achieves the same thing as `[…](…)`. with less noise and repetition.